coalib/coala_delete_orig.py

```diff
@@ -1,21 +1,24 @@
+import logging
 import os
 
-from pyprint.ConsolePrinter import ConsolePrinter
-
 from coalib.output.Logging import configure_logging
-from coalib.output.printers.LogPrinter import LogPrinter
 from coalib.parsing import Globbing
+from coalib.parsing.Globbing import glob_escape
 from coalib.settings.ConfigurationGathering import get_config_directory
 from coalib.settings.Section import Section
-from coalib.parsing.Globbing import glob_escape
 
 
 def main(log_printer=None, section: Section=None):
     configure_logging()
+    return delete_orig(log_printer, section)
+
+
+def delete_orig(log_printer=None, section: Section=None):
+    if log_printer is not None:
+        logging.warning('Using log_printer is deprecated, '
```

Comment:
'is deprecated' ?  It looks like 'is no longer supported' is more correct based on the current code, but ... see next comment.